superset/views.py

```diff
@@ -5,6 +5,7 @@
 
 from datetime import datetime, timedelta
 import json
+import simplejson as sjson
```

Comment:
looks like we import this as `import simplejson as json` in other modules. i find `sjson` looks like a typo to me. either way, being consistent and importing simplejson the same var name seems like a good idea.